py3status/modules/taskwarrior.py

```diff
@@ -38,7 +41,8 @@ def taskWarrior(self):
         def describeTask(taskObj):
             return str(taskObj['id']) + ' ' + taskObj['description']
 
-        task_command = 'task start.before:tomorrow status:pending export'
+        filter = self.filter if self.filter is not None else ''
+        task_command = 'task %s export' % filter
```

Comment:
`if self.filter is not None:` should work with `filter =` `None`, but not `''`.
`if self.filter:` should work with `filter =` `None`, `''`, `False`, `[]`, or `0`

We want...
```
task_command = 'task %s export' % self.task_args \
    if self.task_args else 'task export'
```
or something more readable...
```
    if self.task_args:
        task_command = 'task %s export' % self.task_args
    else:
        task_command = 'task export'
```
```
    task_command = 'task export'
    if self.task_args:
        task_command = 'task %s export' % self.task_args
```